cGame_logic.cpp

```diff
@@ -688,16 +690,16 @@ bool cGame::setupGame() {
 
     game.init(); // Must be first! (loads game.ini file at the end, which is required before going on...)
 
-    logger->logHeader("Dune II - The Maker");
+    logger->logHeader(D2TM_APP_NAME);
```

Comment:
Let's leave the string here, that's easier than using a define.
Also, these defines are available in each and every source file, which goes against the principle of data hinding.